src/official-account/official-account.ts

```diff
@@ -171,9 +179,9 @@ class OfficialAccount extends EventEmitter {
     }
 
     this.accessTokenPayload = {
-      expiresIn : ret.body.expires_in,
-      timestamp : Date.now(),
-      token     : ret.body.access_token,
+      expiresIn: ret.body.expires_in,
+      timestamp: Date.now(),
+      token: ret.body.access_token,
```

Comment:
Could we keep the spaces for better readability? Because I feel that each import should be a separate block visually.

Reply:
Is there any plugins to help me keep code style consistent with you ?

Reply:
Here's what you need!

https://github.com/wechaty/wechaty/issues/745#issuecomment-326113791

Reply:
Awesome to get all of these rules.